### ISSUE TITLE
[EMAGE] Magnitude of FGD

### ISSUE THREAD

**Issue by wubowen416** [PantoMatrix/PantoMatrix#110]
Hi Liu,

In your arxiv paper, Table 4, it says that the reported FGD is x10^-1. Is this correct? Or should it be x10?

When I tested the pretrained AE, the FGD is around 57 between the validation set and gaussian noise. It will be 5.7 after scaled by 0.1, which is similar to the reported performance. This cannot be true.

In addition, the FGD between train set and val set is around 0.52. It will be 5.2 if scaled by 10, which is similar to the reported value for your proposed model.

Appreciate it if you can take a look at this.
Thanks in advance.

**Comment by H-Liu1997** [comment]
Yes, I actually want to say x10, I will revise it in the next version!

Thanks!

**Comment by wubowen416** [comment]
Thank you for your quick reply!
I will close the issue.
